src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -74,6 +90,35 @@ export const TemplatesCatalogDrawerCreateForm: React.FC<TemplatesCatalogDrawerCr
         });
     };
 
+    const onCustomizeVM = () => {
+      k8sCreate<V1Template>({
+        model: ProcessedTemplatesModel,
+        data: template,
+        queryParams: {
+          dryRun: 'All',
+        },
+      }).then((processedTemplate) => {
+        const vmObj = getTemplateVirtualMachineObject(processedTemplate);
+        const updatedVM = produce(vmObj, (vmDraft) => {
+          ensurePath(vmDraft, 'metadata.labels');
+          vmDraft.metadata.name = vmName;
+          vmDraft.metadata.namespace = ns || DEFAULT_NAMESPACE;
+          vmDraft.metadata.labels[LABEL_USED_TEMPLATE_NAME] = processedTemplate.metadata.name;
+          vmDraft.metadata.labels[LABEL_USED_TEMPLATE_NAMESPACE] =
+            processedTemplate.metadata.namespace;
+        });
+        updateVM(updatedVM)
+          .then(() => {
+            history.push(
+              `templatescatalog/customize?name=${template.metadata.name}&namespace=${template.metadata.namespace}`,
+            );
+          })
+          .catch((e) => {
+            console.error(e);
```

Comment:
please try to avoid using console.error() to display errors and follow the pattern of `quickCreateError` state

Reply:
Okay, I'll take a look, thank you! :)